app/lib/dimensionsMiddleware.js

```diff
@@ -0,0 +1,23 @@
+import { Dimensions } from 'react-native';
+
+import { dimensionsWindow, dimensionsScreen } from '../actions/dimensions';
+
+export default () => createStore => (...args) => {
+	const store = createStore(...args);
+
+	const handleDimensionsChange = (nextDimensionState) => {
```

Comment:
Apparently, it's firing twice each `change` event.
It's not an error on your PR, but on RN.
Can you create an if condition to compare previous vs next?
Example: https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/lib/appStateMiddleware.js#L12